README.md

```diff
@@ -6,7 +6,9 @@
 The Etebase TypeScript client API for the web, node and react-native!
 
 
-![GitHub tag](https://img.shields.io/github/tag/etesync/etebase-js.svg)
-![NPM version](https://img.shields.io/npm/v/etebase.svg)
+[![GitHub tag](https://img.shields.io/github/tag/etesync/etebase-js.svg)](https://github.com/etesync/etebase-js/tags)
```

Comment:
Seemed more useful than clicking through to the SVG :shrug:

Reply:
Whoops, yeah, definitely!